met-api/src/met_api/models/engagement.py

```diff
@@ -9,15 +9,16 @@
 
 from .db import db, ma
 from .default_method_result import DefaultMethodResult
-
+from sqlalchemy.dialects.postgresql import JSON, UUID
 
 class Engagement(db.Model):
     """Definition of the Engagement entity"""
     __tablename__ = 'engagement'
 
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     name = db.Column(db.String(50))
-    description = db.Column(db.String(50))
+    description = db.Column(db.Text, unique=False, nullable=False)
+    rich_text_state = db.Column(JSON, unique=False, nullable=False)
```

Comment:
the name rich_text_state is a bit hard to grasp.. we can put the business name against it.. like what exactly are we captuing ;is it a comment from the user or is it long_description ;more of a business name